docs/examples/model_beams.py

```diff
@@ -0,0 +1,47 @@
+from compas.files import OBJ
+from compas.datastructures import Mesh
+from compas_model.viewer import ViewerModel
+from compas_model.elements import Block
+from compas_model.model import Model
+from compas_model.algorithms import Collider
```

Comment:
as mentioned in other PR, this could just be `from compas_model.algorithms import collider` and use `collider.get_collision_pairs` further down

Reply:
implemented to all cases